resources/windows/wix/directorylist.wxs.erb

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <Fragment>
+
+  <%- generate_wix_dirs(project) -%>
```

Comment:
So I can do that, but get_directories is a function inside the project object, so if we do not have access to the project object I'm fairly certain we would not have access to the get_directories functionality.